### PR TITLE
Recipe/ogi kids speech

### PR DESCRIPTION
**Pull request by anyuyay**
## What?

Commit the recipe ogi_kids_speech for ASR of OGI kids' speech corpus.

Also modify CONTRIBUTING.md to make it clearer.

### CODE REVIEW

File: egs2/ogi_kids_speech/asr1/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# RESULTS
```
Comment (sw005320) [inline egs2/ogi_kids_speech/asr1/README.md:1]: Can you add a comment that this is the short-sentence version?
Please explain the details of which portion in the directory you used, how many recordings, hours, etc.
You can also add TODO to deal with the long recording.

File: egs2/ogi_kids_speech/asr1/run.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```
Comment (sw005320) [inline egs2/ogi_kids_speech/asr1/run.sh:1]: Again, please mention that this recipe is the short sentence version

**Review by sw005320 — COMMENTED** [review]
- Please put the model link once you upload it to HF
- The performance is too good. I'm concerned that the test set contamination may happen. Can you check this with @jctian98?

**Comment by anyuyay** [comment]
> * Please put the model link once you upload it to HF
> * The performance is too good. I'm concerned that the test set contamination may happen. Can you check this with @jctian98?

Model uploaded.
Performance is not strange according to the paper for the corpus.
Descriptions for readme.md, run.sh added.

However, the new pull request fails in github code checking (the previous pull request passed the check). I check the test case, it raises warning for "test_multi_speaker_vits_generator_forward[model_dict7]", test/espnet2/asr/decoder/test_whisper_decoder.py, and some other files. I only add several lines of descriptions and comments, so I guess this is not my fault?

**Comment by sw005320** [comment]
Thanks for dealing with them!

> However, the new pull request fails in github code checking (the previous pull request passed the check). I check the test case, it raises warning for "test_multi_speaker_vits_generator_forward[model_dict7]", test/espnet2/asr/decoder/test_whisper_decoder.py, and some other files. I only add several lines of descriptions and comments, so I guess this is not my fault?

This comes from the randomness of CI (if CI assigns to slow machines, they cannot conduct the test within the time limit). I reran the CI. I'll take care of the rest. Once the CI is passed fully, I'll merge this PR.

**Comment by sw005320** [comment]
Thanks, @anyuyay!

**Pull request merged**